recipes/management/README.md

```diff
@@ -10,6 +10,71 @@ This is a simple, powerful way to slice our management recipes. If you are looki
+* Measurement of progress is imperfect ('fog of war')
+* How do we split WINNING vs INCREASING CAPACITY?
+    * It feels like winning is MOVING UP THE CURRENT LOCAL MAX
+    * 'Winning' can be long-term losing. Examples: Google+, TellApart bought by Twitter, Loomis at Google...
+    * So 'winning' has different *resolutions*. E.g. you can win a battle but lose the war. You can win in engineering (ship the code) and lose at product (nobody finds it useful).
+    * Winning is a local definition. There are 'layers' of winning. 
+        * A junior engineer needs to write good code. 
+        * A senior engineers needs to design a scalable system.
+        * A soldier needs to take the next house.
+        * A captain needs to take the next village.
+* What is a PROBLEM?
+    * Three types of problems. 
+        * Inertia 
+        * ?? (things falling apart) 
+        * Power: force in a different direction than you want.
+* What are the TOOLS of winning?
+    * As an engineer:
+        * Writing code: having computers repeatedly solve the problem for you.
+        * Simplifying a design: preventing problems from happening altogether.
+        * Standardization? Making a group of problems all the same problem?
+        * Operational: doing one-off work to solve a problem (e.g. deleting files to free up disk space)
+    * As a manager:
+        * 
+    * As a manager-of-managers:
+        * 
+* For the various layers: your WINNING is their INCREASING CAPACITY?
+    * Teaching an engineer how to write better code is WINNING for an EM. But it's increasing THEIR capacity??
+    * Recursive? It does feels recursive. When you solve a problem, it removes friction/energy being used to tackle that problem, which frees up energy for other problems, meaning you've increased SOMEONE'S capacity to win...?
+    * E.g. Mashgin makes checkout easier. Frees up time for customers --> they can use that time to do more.
+    * ALL TECHNOLOGY is increasing the capacity to win for users of the technology --> is all winning translate to increasing the capacity to win for someone else?
+    * 
+* Mashgin example problems
+    * Need to integrate with P97 
+        * Increases the capacity of our sales team to win. They can close more deals!
+        * If we close one deal, we get more revenue. Increases our capacity to win since we have more $. For the customer, if they are getting value from the product, it increases their capacity to win somehow as well.
+        * Integration would be considered a win. But now assume we do NOT sell any more pilots because of it? Then we haven't actually increased the capacity of anyone, and we wasted resources doing it. So the 'win' ends up being a loss. But how could we know that ahead of time?
+        * Time value of problem solving? ROI? What's the right way to think about this? Increasing capacity is non-deterministic; just because we increased capacity doesn't mean that more wins WILL happen.
```

Comment:
it sounds probabilistic to me - the fact that you have a loaded coin that gives you 60% heads, doesn't mean you'll always get heads... You just have a higher capacity to get them which in the long run translates to more winning...?

Reply:
So in other words, even if you don't get more deals here, you should learn and improve your processes based on this learning and win more in the future?

Reply:
the more I think about it the more useful it may be to show it to more people and get their feedback and examples to test the theory:)